### ISSUE TITLE
help the developpers to detect that there is 2 reducer with same name/prefix

### ISSUE THREAD

**Issue by fabienjuif** [alakarteio/k-redux-factory#79]
Only in development builds:
 - use a global variable to register all tuples `(name, prefix)`
 - detect if there is a collision
 - if there is a collision, print a warning (error ?) message, something like: `[k-redux-factory] You combined two reducers with same name and prefix, we think this is a bug. You should look at 'prefix' option to create your reducer(s)!`

@bpetetot are you ok ?

This is related to #73 that is coming back on a regular basis

**Comment by bpetetot** [comment]
Yes it seems good to me and useful to avoid problems to avoid name collisions

**Comment by fabienjuif** [comment]
@bpetetot do you know how to handle dev/prod build from a library perspective ?

**Comment by bpetetot** [comment]
In ReactJS, they build 2 differents bundles: `react.development.js` and `react.production.js`
With the following `index.js` file:

```js
'use strict';

if (process.env.NODE_ENV === 'production') {
  module.exports = require('./cjs/react.production.min.js');
} else {
  module.exports = require('./cjs/react.development.js');
}
```

**Comment by fabienjuif** [comment]
Hmm ok, so we have to build 2 times the libs:
 - `NODE_ENV=production yarn build`
 - `NODE_ENV=development yarn build`

And add this index.js manually

**Comment by bpetetot** [comment]
I think you can directly build dev and prod bundles through the rollup config. So you will not have to call 2 times the `yarn build` command.

**Comment by fabienjuif** [comment]
I move the lib to microbundle
